src/Compilers/CSharp/Portable/Symbols/LexicalSortKey.cs

```diff
@@ -51,6 +51,10 @@ public int Position
         public static readonly LexicalSortKey SynthesizedCtor = new LexicalSortKey() { _treeOrdinal = int.MaxValue, _position = int.MaxValue - 1 };
         public static readonly LexicalSortKey SynthesizedCCtor = new LexicalSortKey() { _treeOrdinal = int.MaxValue, _position = int.MaxValue };
 
+        // Put Record Equals right before the above
```

Comment:
Please order these property declarations like you actually want them ordered, top to bottom. It makes it much easier to read.